src/planwise/endpoint/facilities.clj

```diff
@@ -2,18 +2,27 @@
   (:require [planwise.boundary.facilities :as facilities]
             [planwise.boundary.maps :as maps]
             [compojure.core :refer :all]
+            [clojure.string :as string]
             [buddy.auth.accessrules :refer [restrict]]
             [buddy.auth :refer [authenticated?]]
             [ring.util.response :refer [response]]))
 
-(defn facilities-criteria [type region]
-  {:region (if region (Integer. region) nil)
-   :types (if type (map #(Integer. %) (vals type)) nil)})
+(defn- facilities-criteria [{:keys [type region bbox excluding]}]
+  {:region (when region (Integer. region))
```

Comment:
Alternatively, you can use `(some-> region Integer.)` which *may* be more readable/idiomatic